cedar-policy/src/frontend/is_authorized.rs

```diff
@@ -557,19 +564,7 @@ impl RecvdSlice {
             template_instantiations,
         } = self;
 
-        let policy_set = match policies {
-            PolicySpecification::Concatenated(policies) => match PolicySet::from_str(&policies) {
```

Comment:
This parsing code, and the `parse_policy_set_from_individual_policies` function below were copied verbatim to `util.rs`.